ui/app/components/app/selected-account/index.scss

```diff
@@ -37,6 +37,8 @@
     padding: 6px 1px;
     border-radius: 10px;
     cursor: pointer;
+    width: 100%;
+    background-color: unset;
```

Comment:
q: What issue does this resolve with the styling?

Reply:
Without these styles the button is essentially only as large as it needs to be, thus this would change layout. Additionally, we don't want the native OS button background color.